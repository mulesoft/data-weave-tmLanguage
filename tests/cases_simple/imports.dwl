import * from dw::core::Object
import spread from dw::core::Object
import spread as SPREAD from dw::core::Object
import spread as SPREAD1, b, C, a as D from dw::core::Object
import dw::core::Object as OBJEKT
---
1