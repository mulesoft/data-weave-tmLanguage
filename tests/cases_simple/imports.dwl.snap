>import * from dw::core::Object
#^^^^^^ source.data-weave storage.type.dw
#      ^ source.data-weave
#       ^ source.data-weave keyword.operator.arithmetic.dw
#        ^ source.data-weave
#         ^^^^ source.data-weave storage.type.dw
#             ^ source.data-weave
#              ^^ source.data-weave entity.name.variable.dw
#                ^ source.data-weave keyword.operator.declaration.dw
#                 ^ source.data-weave keyword.operator.declaration.dw
#                  ^^^^ source.data-weave entity.name.variable.dw
#                      ^ source.data-weave keyword.operator.declaration.dw
#                       ^ source.data-weave keyword.operator.declaration.dw
#                        ^^^^^^ source.data-weave entity.name.variable.dw
>import spread from dw::core::Object
#^^^^^^ source.data-weave storage.type.dw
#      ^ source.data-weave
#       ^^^^^^ source.data-weave entity.name.variable.dw
#             ^ source.data-weave
#              ^^^^ source.data-weave storage.type.dw
#                  ^ source.data-weave
#                   ^^ source.data-weave entity.name.variable.dw
#                     ^ source.data-weave keyword.operator.declaration.dw
#                      ^ source.data-weave keyword.operator.declaration.dw
#                       ^^^^ source.data-weave entity.name.variable.dw
#                           ^ source.data-weave keyword.operator.declaration.dw
#                            ^ source.data-weave keyword.operator.declaration.dw
#                             ^^^^^^ source.data-weave entity.name.variable.dw
>import spread as SPREAD from dw::core::Object
#^^^^^^ source.data-weave storage.type.dw
#      ^ source.data-weave
#       ^^^^^^ source.data-weave entity.name.variable.dw
#             ^ source.data-weave
#              ^^ source.data-weave keyword.operator.dw
#                ^ source.data-weave
#                 ^^^^^^ source.data-weave entity.name.variable.dw
#                       ^ source.data-weave
#                        ^^^^ source.data-weave storage.type.dw
#                            ^ source.data-weave
#                             ^^ source.data-weave entity.name.variable.dw
#                               ^ source.data-weave keyword.operator.declaration.dw
#                                ^ source.data-weave keyword.operator.declaration.dw
#                                 ^^^^ source.data-weave entity.name.variable.dw
#                                     ^ source.data-weave keyword.operator.declaration.dw
#                                      ^ source.data-weave keyword.operator.declaration.dw
#                                       ^^^^^^ source.data-weave entity.name.variable.dw
>import spread as SPREAD1, b, C, a as D from dw::core::Object
#^^^^^^ source.data-weave storage.type.dw
#      ^ source.data-weave
#       ^^^^^^ source.data-weave entity.name.variable.dw
#             ^ source.data-weave
#              ^^ source.data-weave keyword.operator.dw
#                ^ source.data-weave
#                 ^^^^^^^ source.data-weave entity.name.variable.dw
#                        ^^ source.data-weave
#                          ^ source.data-weave entity.name.variable.dw
#                           ^^ source.data-weave
#                             ^ source.data-weave entity.name.variable.dw
#                              ^^ source.data-weave
#                                ^ source.data-weave entity.name.variable.dw
#                                 ^ source.data-weave
#                                  ^^ source.data-weave keyword.operator.dw
#                                    ^ source.data-weave
#                                     ^ source.data-weave entity.name.variable.dw
#                                      ^ source.data-weave
#                                       ^^^^ source.data-weave storage.type.dw
#                                           ^ source.data-weave
#                                            ^^ source.data-weave entity.name.variable.dw
#                                              ^ source.data-weave keyword.operator.declaration.dw
#                                               ^ source.data-weave keyword.operator.declaration.dw
#                                                ^^^^ source.data-weave entity.name.variable.dw
#                                                    ^ source.data-weave keyword.operator.declaration.dw
#                                                     ^ source.data-weave keyword.operator.declaration.dw
#                                                      ^^^^^^ source.data-weave entity.name.variable.dw
>import dw::core::Object as OBJEKT
#^^^^^^ source.data-weave storage.type.dw
#      ^ source.data-weave
#       ^^ source.data-weave entity.name.variable.dw
#         ^ source.data-weave keyword.operator.declaration.dw
#          ^ source.data-weave keyword.operator.declaration.dw
#           ^^^^ source.data-weave entity.name.variable.dw
#               ^ source.data-weave keyword.operator.declaration.dw
#                ^ source.data-weave keyword.operator.declaration.dw
#                 ^^^^^^ source.data-weave entity.name.variable.dw
#                       ^ source.data-weave
#                        ^^ source.data-weave keyword.operator.dw
#                          ^ source.data-weave
#                           ^^^^^^ source.data-weave entity.name.variable.dw
>---
#^^^ source.data-weave keyword.operator.body-marker.dw
>1
#^ source.data-weave constant.numeric.dw